src/components/Button/ButtonAddPostTopic.js

```diff
@@ -40,7 +41,7 @@ const styles = StyleSheet.create({
     bottom: normalize(30),
     right: 0,
     zIndex: 99,
-    backgroundColor: '#0000'
+    backgroundColor: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing the hardcoded color value with a dynamic one from the theme object is a good practice for maintainability and consistency.
```diff
-    backgroundColor: '#0000'
+    backgroundColor: COLORS.black
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->